test/Logger/LoggerTest.php

```diff
@@ -2,29 +2,29 @@
 
 namespace DvsaApplicationLoggerTest\Logger;
 
-use AccountApi\Service\TokenService;
 use DvsaApplicationLogger\Factory\LoggerFactory;
 use DvsaApplicationLogger\Helper\SapiHelper;
+use DvsaApplicationLogger\Interfaces\MotIdentityInterface;
+use DvsaApplicationLogger\Interfaces\MotIdentityProviderInterface;
 use DvsaApplicationLogger\Log\Logger;
 use DvsaApplicationLogger\Log\SystemLogLogger;
 use DvsaApplicationLogger\Processor\ReplaceTraceArgsProcessor;
 use DvsaApplicationLogger\Log\FilteredStackTrace;
-
-use DvsaCommon\Auth\MotIdentity;
-use DvsaCommon\Auth\MotIdentityProvider;
+use DvsaApplicationLogger\TokenService\TokenServiceInterface;
 use Laminas\Stdlib\SplPriorityQueue;
 use Interop\Container\ContainerInterface;
 use PHPUnit\Framework\TestCase;
+use Laminas\Log\Writer\Mock;
```

Comment:
I would suggest custom name for this mock so there will not be confusion.
```suggestion
use Laminas\Log\Writer\Mock as WriterMock;
```

Reply:
Done